codes/classical/q-ary_digits/maximum_rank_distance.yml

```diff
@@ -0,0 +1,29 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: maximum_rank_distance
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Maximum rank distance code'
```

Comment:
Maximum-rank code
By the way, why did you decide to make this a separate entry? Is it a well-known enough code that it needs to be separated from rank code?

Reply:
Rank code uses rank metric instead of Hamming distance and just states basic definitions, maximum rank code has a construction of optimal codes